src/components/ButtonPanel.module.css

```diff
@@ -0,0 +1,7 @@
+.ButtonPanel {
```

Comment:
buttonpanel should be styled with flexbox ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)